### DISCUSSION TITLE
Bug report: filename qualifier isn't working

### DISCUSSION THREAD

**Discussion by Avivhdr** [community/community#9492] [Other Feature Feedback, Questions, & Ideas] [answered]
[filename](https://docs.github.com/en/search-github/searching-on-github/searching-code#search-by-filename) qualifier isn't working

**Looking for files named `gitignore` in the old search:**
![image](https://user-images.githubusercontent.com/23265867/147848389-6627e1a7-fdfd-4a6b-b939-54d9ba424f48.png)

**Looking for files named `gitignore` in the new search:**
![image](https://user-images.githubusercontent.com/23265867/147848398-86f5415a-c05b-489c-b44a-d14551798b03.png)

**Comment by gorzell** [comment, marked as answer]
I think you have run into a couple of different issues here, the first is that `filename` isn't a supported qualifier in the new search (as the error notes).  You can find the [supported qualifiers on the syntax page](https://cs.github.com/about/syntax), and the one that you want is [`path`](https://cs.github.com/about/syntax#path).  Next, globs patterns are also not supported, so you will need to use regex syntax instead.  So the search you gave above should actually be `org:tesla path:/.*gitignore.*/`.

**Reply by Avivhdr** [reply]
Hi @gorzell 
Thank you!
I was not aware of `path`
👍🏼
